java/core/src/test/org/apache/orc/impl/TestReaderImpl.java

```diff
@@ -527,4 +527,37 @@ public void testWithoutCompressionBlockSize() throws IOException {
       }
     }
   }
+
+  @Test
+  public void testSargSkipPickupGroupWithoutIndex() throws IOException {
+    Configuration conf = new Configuration();
+    Path[] paths = new Path[] {
+        new Path(workDir, "TestOrcFile.testSargSkipPickupGroupWithoutIndexCPlusPlus.orc"),
+        new Path(workDir, "TestOrcFile.testSargSkipPickupGroupWithoutIndexJava.orc"),
```

Comment:
Please add some descriptions of the test file columns.

Reply:
Good catch! Done. Thanks @guiyanakuang